### DISCUSSION TITLE
Unable to update zigbee2mqtt on hass os 5.13

### DISCUSSION THREAD

**Discussion by shsurury** [Koenkk/zigbee2mqtt#7478] [General]
Hi,

I have installed zifbee2mqtt on HASS os 5.13 with supervisor Core 2021.5.4 however it is not updating to latest version 1.18.3. It is currently stuck to 1.18.1-1. I have also got auto update turned on but no luck. can someone please help.

**Comment by Koenkk** [comment]
The addon has moved to a new repo and for some people it doesn't seem to update automatically. Uninstall the old addon and install the new one will fix it: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt#installation

**Reply by shsurury** [reply]
Thanks Koenkk for your help. All sorted now.

**Reply by wojciechczyz** [reply]
Thats great news. Just one question, can I do this without loosing all devices in HA? their entity names are used in automations etc.

**Comment by shsurury** [comment]
I was able to retain all my devices by changing the PID but no guarantee

**Comment by Nina-Syrina** [comment]
For me, this was pretty easy. 
I installed the new version of zigbee2mqtt on my HA. Then copied the configuration (in the web gui) from the old one to the new one. Turned my old version off. And then started the new version. 

I had to remove the side bar in the old add-on info page and turn it on at the new info page... (that threw me off for a bit because I thought everything was broken and I was close to panicking lol.)

Though my automations and devices just continued to run. I then even updated my Home Assistance instance (because I love to live on the risky side of life. :D) 

Works like a charm so far. <3 
And I hope it stays that way.
